deploy/group_vars/all.yml

```diff
@@ -92,7 +92,7 @@ k8s_container_port: 8000
 k8s_container_image: "{{ RepositoryURL }}"
 k8s_container_image_pull_policy: Always
 k8s_container_replicas: 2
-k8s_migrations_enabled: true
+k8s_migrations_enabled: false
```

Comment:
I think this was changed temporarily for a deploy; could we change it back to "true", so that migrations run on future deploys?